lib/process-services/src/lib/process-list/services/process-filter.service.ts

```diff
@@ -87,8 +85,7 @@ export class ProcessFilterService {
     getProcessFilterByName(filterName: string, appId?: number): Observable<FilterProcessRepresentationModel> {
         return from(this.callApiProcessFilters(appId))
             .pipe(
-                map((response: any) => response.data.find((filter) => filter.name === filterName)),
-                catchError((err) => this.handleProcessError(err))
+                map((response: any) => response.data.find((filter) => filter.name === filterName))
```

Comment:
We can remove `any` here right?

Reply:
removed and fixed the issue with incorrect return type :)